resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -2,7 +2,7 @@
 jruby-puppet: {
     # Where the puppet-agent dependency places puppet, facter, etc...
     # Puppet server expects to load Puppet from this location
-    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
+    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby, /opt/puppetlabs/puppet/cache/lib]
```

Comment:
@cprice404 good point about passenger. Prior to 4.0, we set `vardir` in [`config.ru`](https://github.com/puppetlabs/puppet/blob/3.7.5/ext/rack/config.ru#L18) so passenger and the agent shared the same vardir:

```
ARGV << "--vardir"  << "/var/lib/puppet"
```

But in 4.x, passenger is overriding `vardir` to `"/opt/puppetlabs/server/data/puppetmaster"` and will have the same problem as this.